src/commands/unity-symbols/README.md

```diff
@@ -0,0 +1,47 @@
+## Overview
+
+To deobfuscate and symbolicate errors and crashes, upload IL2CPP mappings and iOS dSYMs to Datadog.
```

Comment:
Can you add this new command in the main README as well?

(In the usage section: https://github.com/DataDog/datadog-ci?tab=readme-ov-file#usage)

Reply:
@fuzzybinary and also add the command to the `CODEOWNERS` file 😉